airflow/providers/google/cloud/transfers/gcs_to_bigquery.py

```diff
@@ -322,101 +345,29 @@ def execute(self, context: Context):
             if self.schema_object and self.source_format != "DATASTORE_BACKUP":
                 schema_fields = json.loads(gcs_hook.download(self.bucket, self.schema_object).decode("utf-8"))
                 self.log.info("Autodetected fields from schema object: %s", schema_fields)
```

Comment:
This logging statement is not correct. Nothing got *autodetected* here, we just loaded the explicitly specified schema.

Reply:
Sure, I have changed the output